public/src/js/widgets/fetch-latest-packages.js

```diff
@@ -19,7 +19,12 @@ export default class extends Extension {
                 isHidden: this.baseModel.priority === 'training'
             }
         })
-        .then(response => this.baseModel.latestPackages(response.results))
+        .then(result => {
+            return this.baseModel.latestPackages(result.response.results.map(result => {
+                result.name = result.packageName;
```

Comment:
you shouldn't modify objects inside a map, just return a new object with the things you need